install_config/http_proxies.adoc

```diff
@@ -44,8 +44,8 @@ endif::[]
 The `NO_PROXY` environment variable lists all of the {product-title}
 components and all IP addresses that are managed by {product-title}.
 
-`NO_PROXY` accepts a comma-separated list of hosts, IP addresses, or IP ranges
-in CIDR format:
+In all cases except for Docker, `NO_PROXY` accepts a comma-separated list of
+hosts, IP addresses, or IP ranges in CIDR format:
 
 For master hosts::
 * Node host name
```

Comment:
Can we add here  to the list for masters. 
* Ip address of etcd hosts. 

This was not address in bug. Just trying knock this out while we are changing this doc. Thank you
https://bugzilla.redhat.com/show_bug.cgi?id=1458660